frontend/src/scenes/pipeline/hogfunctions/hogFunctionConfigurationLogic.tsx

```diff
@@ -833,6 +839,15 @@ export const hogFunctionConfigurationLogic = kea<hogFunctionConfigurationLogicTy
             (s) => [s.hogFunction, s.template],
             (hogFunction, template) => template?.mapping_templates ?? hogFunction?.template?.mapping_templates ?? [],
         ],
+
+        usesGroups: [
+            (s) => [s.configuration],
+            (configuration) => {
+                // NOTE: Bit hacky but works good enough...
+                const configStr = JSON.stringify(configuration)
+                return configStr.includes('{groups')
```

Comment:
We might want to check for `groups.` instead to make sure we catch these expressions as well:

```
trim(groups.company.name)
```

```suggestion
                // NOTE: Bit hacky but works good enough...
                const configStr = JSON.stringify(configuration)
                return configStr.includes('groups.') || configStr.includes('{groups}')
```